docs/FAQ/how-to-deploy-bold-bi-in-eks-using-application-load-balancer.md

```diff
@@ -0,0 +1,220 @@
+          stringData:
+            # Type of database server can be used for configuring the Bold BI. Eg: mssql, mysql and postgresql
+            BOLD_SERVICES_DB_TYPE: "postgresql"
+
+            # Name of the Database Server
+            BOLD_SERVICES_DB_HOST: "localhost"
+
+            # The system will use the following default port numbers based on the database server type.
+            # PostgrSQL – 5432 and MySQL -3306
+            # Please specify the port number for your database server if it is configured on a different port.
+            # For MS SQL Server, this parameter is not necessary.
+            BOLD_SERVICES_DB_PORT: "5432"
+
+            # Username for the database server
+            # Please refer to this documentation for information on the user's permissions.
+            # https://help.boldbi.com/embedded-bi/faq/what-are-the-database-permissions-required-to-set-up-bold-bi-embedded/
+            BOLD_SERVICES_DB_USER: "boldbi@boldbi-docker"
+
+            # The database user's password
+            BOLD_SERVICES_DB_PASSWORD: "F8o:z$jasoKkel"
+
+            # If the database name is not specified, the system will create a new database called bold services.
+            # If you specify a database name, it should already exist on the server.
+            BOLD_SERVICES_DB_NAME: ""
+
+            # For PostgreSQL DB Servers, this is an optional parameter.
+            # The system will use the database name postgres by default.
+            # If your database server uses a different default database, please provide it here.
+            BOLD_SERVICES_POSTGRESQL_MAINTENANCE_DB: "postgres"
+
+            # If your database server requires additional connection string parameters, include them here
+            # Connection string parameters can be found in the official document.
+            # My SQL: https://dev.mysql.com/doc/connector-net/en/connector-net-8-0-connection-options.html
+            # PostgreSQL: https://www.npgsql.org/doc/connection-string-parameters.html
+            # MS SQL: https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.connectionstring
+            # Note: A semicolon(;) should be used to separate multiple parameters.
+            BOLD_SERVICES_DB_ADDITIONAL_PARAMETERS: ""
+          ---
+        </details>
+           
+    * If you need to use the **Bing Map** widget feature, enter the value for `widget_bing_map_enable` environment variable as `true` and the API key value for `widget_bing_map_api_key`.
+       
+       ![Enable-Bingmap](../images/deploy/eks/bingmap_enable.png)
+
+11. If you have a DNS to map with the application, proceed to the next steps, otherwise, skip to **Step 13**. 
+
+12. Download the `ingress_alb.yaml` file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/main/deploy/deploy/ingress_alb.yaml) Uncomment the host value and replace your DNS hostname with `example.com` in ingress_alb.yaml file in line **18**.
```

Comment:
12. Download the `ingress_alb.yaml` file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/main/deploy/deploy/ingress_alb.yaml). Uncomment the host value and replace your DNS hostname with `example.com` in the ingress_alb.yaml file in line **18**.

Reply:
Concern addressed